.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: required
+language: rust
```

Comment:
Best to also add https://github.com/sdroege/gobject-subclass/blob/1f0334f275b8f5e987f9b57474c5cf1783665eda/.travis.yml#L4-L5 to speed things up